web/src/pages/[...slug].page.tsx

```diff
@@ -189,20 +194,24 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
 		}
 		if ("notFound" in storyResult) return storyResult;
 
-		const siblingPages = [];
+		const parentID = storyResult.story?.parent_id as number;
+		const isRootPage = storyResult.story?.is_startpage;
 
 		const component = storyResult.story?.content?.component;
-		// TODO: Use the Storyblok Links API to build a map of sibling & optionally child pages
+		let tree: ExtendedSBLink[] = [];
+
 		if (component === "infoPage") {
-			siblingPages.push(...["page1", "page2"]);
+			tree = await buildTree(parentID, slug, isRootPage);
+			// TODO: move out of catchall page; would need API route as GSSP is not allowed in components whilst using pages router
```

Comment:
all looks good to me, but lets not forget to move this tree logic out of the page at some point